v2/svid/x509svid/svid.go

```diff
@@ -47,7 +50,7 @@ func Load(certFile, keyFile string) (*SVID, error) {
 // Parse parses the X509-SVID from PEM blocks containing certificate and key
 // bytes. The certificate must be one or more PEM blocks with ASN.1 DER. The
 // key must be a PEM block with PKCS#8 ASN.1 DER.
-func Parse(certBytes, keyBytes []byte) (*SVID, error) {
+func Parse(certBytes, keyBytes []byte, opts ...optional.SVIDOption) (*SVID, error) {
```

Comment:
Now that 2.0 has shipped, I don't think we should be changing the signature of exported functions unless the benefit is very clear. In this case, I don't really have a good sense on how hints will be leveraged enough to understand if it's really a hardship for someone to call Parse and then set the hint on the returned SVID v.s. having a way to set the SVID hint while parsing. I'd probably lean towards not providing these until we fully understand how folks will want to interact with the hint.

Reply:
Makes sense, I will keep things simple here